app/models/orphan.rb

```diff
@@ -134,11 +136,15 @@ def sponsored_siblings_does_not_exceed_siblings_count
   end
 
   def default_sponsorship_status_unsponsored
-    self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    if new_record?
+      self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    end
   end
 
   def default_orphan_status_active
-     self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    if new_record?
+      self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    end
```

Comment:
I was thinking that this would be cleaner:
```ruby
after_initialize :set_defaults

def set_defaults
  return unless new_record?
  self.orphan_sponsorship_status ||= ...
  self.orphan_status ||= ...
  self.priority ||= ...
end
```
No need to override anything. `after_initialize` currently calls 3 methods that set defaults for a new instance, instead it could be calling one.

This is a minor point anyway.